### ISSUE TITLE
Crash when having glowing effect

### ISSUE THREAD

**Issue by liujaxu** [Leawind/Third-Person#112]
[crash-2024-06-19_20.41.11-client.txt](https://github.com/user-attachments/files/15901016/crash-2024-06-19_20.41.11-client.txt)
## When
The Minecraft version is 1.20.1 Fabric with only Leawind's Thrid Person mod and architectury API lib. When the player has glowing effect in 3rd person perspective, the game crashes in a few seconds.

## Crash report
Crash report is given.

**Comment by Leawind** [comment]
fixed in ff3dac80d90e8210705d26e1e27f96e2f23afa38

**Comment by Leawind** [comment]
Key words:

```log

java.lang.ClassCastException: class net.minecraft.class_4618 cannot be cast to class net.minecraft.class_4597$class_4598 (net.minecraft.class_4618 and net.minecraft.class_4597$class_4598 are in unnamed module of loader net.fabricmc.loader.impl.launch.knot.KnotClassLoader @3b2da18f)
	at net.minecraft.class_761.handler$zok000$leawind_third_person$renderEntity_tail(class_761.java:4443)
	at net.minecraft.class_761.method_22977(class_761.java:1575)
	at net.minecraft.class_761.method_22710(class_761.java:1324)
	at net.minecraft.class_757.method_3188(class_757.java:1110)
	at net.minecraft.class_757.method_3192(class_757.java:880)
	at net.minecraft.class_310.method_1523(class_310.java:1219)
	at net.minecraft.class_310.method_1514(class_310.java:802)
	at net.minecraft.client.main.Main.main(Main.java:250)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:71)
	at oolloo.jlw.Wrapper.main(Wrapper.java:51)

```
